src/include/sof/dma.h

```diff
@@ -168,6 +171,8 @@ struct dma_ops {
 
 	int (*get_data_size)(struct dma *dma, int channel, uint32_t *avail,
 			     uint32_t *free);
+
+	int (*get_attribute)(struct dma *dma, uint32_t type, uint32_t *value);
```

Comment:
Please split this commit into two separate:
1. Adding ops to dma.
2. Using ops in host.

Reply:
ok